client/public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Do we need to use this link? We can import icons from material-ui-icons

Reply:
No we don't need it I'll remove it I was using this first but then i saw there was the material-ui-icons package